doc/source/whatsnew/v1.0.0.rst

```diff
@@ -29,7 +29,7 @@ Enhancements
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
--
+- String support for paramater partition_cols in the :func:`pandas.to_parquet` (:issue:`27117`)
```

Comment:
```suggestion
- String support for paramater ``partition_cols`` in the :func:`pandas.to_parquet` (:issue:`27117`)
```

I would also try to make it a bit clearer to say that a string for a single column name is now also accepted instead of a list

Reply:
do we really want to accept a sting here (and not a list)? i think we would rather be strict about this; though i suppose this is convenient as a partition col is often a single str